adapters/handlers/rest/configure_api.go

```diff
@@ -750,12 +751,15 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 			}
 		}, appState.Logger)
 	}
-	enterrors.GoWrapper(
-		func() {
-			ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-			defer cancel()
-			backupScheduler.CleanupUnfinishedBackups(ctx)
-		}, appState.Logger)
+	if entcfg.Enabled(os.Getenv("CLEANUP_UNFINISHED_BACKUPS")) {
```

Comment:
let's change the name of this env variable to `ENABLE_CLEANUP_UNFINISHED_BACKUPS`